nrt/data/IEEE14/MC/reference/aggregatedResults.xml

```diff
@@ -4,19 +4,15 @@
     <scenarioResults id="DisconnectLine" status="CONVERGENCE"/>
     <scenarioResults id="DisconnectGroup" status="CONVERGENCE"/>
   </loadIncreaseResults>
-  <loadIncreaseResults loadLevel="50" status="CONVERGENCE">
+  <loadIncreaseResults loadLevel="63" status="CONVERGENCE">
```

Comment:
How come that this dev is modifying the MC results?

Reply:
Simulation results are contained in a map (I replaced the vector results_ by a map), so simulation results are sorted. That's why they are also sorted in the aggregatedResults.xml and that duplicates disappeared. The rest of the results remained unchanged.